.ci/opensearch/docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
       - discovery.type=single-node
       - bootstrap.memory_lock=true
       - SECURE_INTEGRATION=${SECURE_INTEGRATION:-false}
+      - OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123!
```

Comment:
@DarshitChanpura with OS 2.12 and Up, would this be enough to set the admin password? Does the plugin just grab what stored in `OPENSEARCH_INITIAL_ADMIN_PASSWORD`, check if it's strong enough, and then assign it as password for `admin` user?

Reply:
I also saw this in the security plugin repo: https://github.com/opensearch-project/security/pull/3843/files
Does that mean that if we set `initialAdminPassword` to `myStrongPassword123!`, it will also work for older version of OS?